detox/test/e2e/t-pasteboard.js

```diff
@@ -10,29 +10,29 @@ describe('Pasteboard', () => {
   it('should have stringTextInput', async () => {
     await element(by.id('stringValueInput')).typeText('exampleString\n');
     await element(by.id('CheckButton')).tap();
-    await expect(device).pasteboardToHaveString('exampleString');
+    await expect(await device.pasteboard()).toHaveString('exampleString');
     await element(by.id('backButton')).tap();
   });
 
   it('should have imageValue', async () => {
     await element(by.id('testImageValue')).tap()
     await element(by.id('CheckButton')).tap()
-    await expect(device).pasteboardToHaveImage();
+    await expect(await device.pasteboard()).toHaveImage();
     await element(by.id('backButton')).tap()
   });
 
   it('should have urlValue', async () => {
     await element(by.id('testURLValue')).tap();
     await element(by.id('testURLValue')).typeText('exampleURL\n');
```

Comment:
is that a URL or string? this test seems weird. This is not a URL, yet pasteboard returns true for `toHaveURL`